<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>411</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_OpenFile">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>640</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_xMove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x_move</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y_move</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_yMove">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z_move</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_zMove">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_xTurn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x_turn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y_turn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z_turn</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_yTurn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_zTurn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_scale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reprint">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Reprint</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Projection type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_projection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Central</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Parallel</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Background color </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_backgroundcolor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>background color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_linetype">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Solid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dashed</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Line color </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_linecolor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Line color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Line type</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_linesize">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>line size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertex type</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_vertexsize">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>vertex size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertex color </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_vertextype">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Square</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_vertexcolor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertex color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_saveFile">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>650</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save file</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_filetype">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>610</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.bmp</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.jpeg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.gif</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>File type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_defaultsettings">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Default settings</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
